elasticsearch/repository.sls

```diff
@@ -1,26 +1,53 @@
-{% from "elasticsearch/map.jinja" import elasticsearch with context %}
-{% set os_family = grains['os_family'] %}
+{% from "elasticsearch/map.jinja" import elasticsearch, elasticsearch_repo with context %}
+{% set os_family = salt.grains.get('os_family') %}
+{% set osfullname = salt.grains.get('osfullname') %}
 
 {% if os_family == 'RedHat' %}
 install_elasticsearch_gpg_key:
   cmd.run:
-    - name: rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch
+    - name: rpm --import {{ elasticsearch.gpg-key }}
     - require_in:
         - pkgrepo: configure_elasticsearch_package_repo
 {% endif %}
 
-{% for name, version in elasticsearch.products.items() %}
```

Comment:
We don't want to get rid of the for loop here because that is what allows us to support Kibana, Logstaash, etc. for pre 5.x releases.